blocks/tiles/tiles.js

```diff
@@ -0,0 +1,110 @@
+export default async function decorate(block) {
+  // Create containing div of three tiles (one big, two small)
+  const tileContainer = document.createElement('div');
+  tileContainer.className = 'tiles-container';
+
+  const data = await new Promise((resolve, reject) => {
+    const urls = [...block.children].map((row) => {
+      const path = new URL(row.firstElementChild.firstElementChild.text).pathname;
+      return `https://admin.hlx.page/index/hlxsites/petplace/main/${path}`;
```

Comment:
*nitpicking* you probably have a double slash in the resulting URL
```suggestion
      return `https://admin.hlx.page/index/hlxsites/petplace/main${path}`;
```